docs/man-pages/condor_test_token.rst

```diff
@@ -24,10 +24,13 @@ Description
 -----------
 
 *condor_test_token* generates a temporary signing key, adds it to the
-local SciTokens cache for the given issuer, and creates a short-lived
-token signed by the key.
+local SciTokens cache for the given issuer, creates a short-lived
+token signed by the key, and prints the token to stdout.
 Local HTCondor daemons will treat this token like any reguluar token
-generated by the given issuer.
+generated by the given issuer for a short period of time (one hour).
+
+If the HTCondor daemons were started as root, then the tool must be
+run as the condor user.
```

Comment:
Our default installation the condor account has nologin for a shell. Perhaps, we should give an example: `su -s /bin/bash -c condor_test_token`

I tried that on a fresh AlmaLinux 9 Docker container, and it said: 
`Traceback (most recent call last):
  File "/usr/bin/condor_test_token", line 11, in <module>
    from cryptography.hazmat.primitives.asymmetric import ec
ModuleNotFoundError: No module named 'cryptography'
`

Reply:
We should also look at having the script perform the important actions as user condor if it's started as root.
I had forgotten that the script relies on a python module that's not installed by default. I'll make a ticket for both of these points.

Reply:
Given that we have made a ticket to improve `condor_test_token`, I'll approve the documentation as is.